common-practices-tools/contribution/contrib-first.md

```diff
@@ -0,0 +1,29 @@
+# Contrib First
+
+Whenever we are building something that could be of use by more than one project or client, we build it as contributed work first, if our contract and security concerns allow for it. Contrib First is an approach we take with both patches to FOSS software and new contributions.
+
+## Rationale for contrib first
+
+-   **Fiscal responsibility** - Building it and contributing it means that other government agencies will never have to pay to build the same thing twice. This helps agencies comply with Federal Source Code Policy: Achieving Efficiency, Transparency, and Innovation through Reusable and Open Source Software [OMB Memorandum M-16-21](https://obamawhitehouse.archives.gov/sites/default/files/omb/memoranda/2016/m_16_21.pdf)
+-   **Reusability** - CivcActions other clients can benefit from work that was already done.
```

Comment:
What about broadening this to non-federal, non-CivicActions users - e.g. "CivicActions other clients and the public at large can benefit"

Reply:
Nice.   Done.